components/FormComponents.js

```diff
@@ -47,6 +65,7 @@ function Input({
         error={error}
         onChange={(event) => handleChange(event)}
         onKeyDown={(event) => handleKeyPress(event)}
+        defaultValue={value ? value : ""}
```

Comment:
you can simplify this:
```suggestion
        defaultValue={value || ""}
```